lib/scenarios.rb

```diff
@@ -13,6 +13,7 @@ class Scenarios < BaseUtil
   PLANS_UQHP_FAMILY = 'plans_for_uqhp_family'
   PLANS_CSR_FAMILY = 'plans_for_family_receiving_csr'
   RIDP = 'ridp'
+  USER_EXISTENCE = 'user_existence'
 
```

Comment:
see above, I think we'll want `user_exists_in_enroll_IVL`, `user_exists_in_enroll_SHOP`, and `user_not_in_enroll`  cases